sdk/src/test/scala/io/horizen/account/fixtures/AccountBlockHeaderFixture.scala

```diff
@@ -53,7 +53,7 @@ trait AccountBlockHeaderFixture {
     val unsignedHeader = AccountBlockHeader(
       blockVersion,
       bytesToId(parentId),
-      random.nextInt(100) + 10000,
+      random.nextInt(100) + 90000,
```

Comment:
The timestamp of the  block header was older than genesis and therefore caused issues.